core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/V1WorkspaceMigrationMonitor.scala

```diff
@@ -197,6 +213,24 @@ object V1WorkspaceMigrationMonitor
         .update((tmpBucketName.value.some, Timestamp.valueOf(LocalDateTime.now).some))
     ))
   }
+
+  final def claimAndConfigureNewGoogleProject(migrationAttempt: V1WorkspaceMigrationAttempt,
+                                              workspaceService: WorkspaceService,
+                                              workspace: Workspace,
+                                              billingProject: RawlsBillingProject
+                                             ): Future[(GoogleProjectId, GoogleProjectNumber, WriteAction[Unit])] =
```

Comment:
Can we stick to either `IO` or `Future` in this class?

Reply:
Sadly this isn't possible owing to IO and Future not implementing the same set of type classes. I'll coerce `Future` into `IO` so there's some consistent return types at least